docker/bragi/Dockerfile

```diff
@@ -39,17 +35,12 @@ RUN --mount=type=cache,target=/home/mimirsbrunn/target \
 # Target image
 # ============
 
-FROM debian:buster-slim
+FROM rust:1.60-alpine3.15
 
 WORKDIR /srv
 
-ENV DEBIAN_FRONTEND noninteractive
-
-RUN apt-get update
 # curl required for healthchecks
-RUN apt-get install -y curl libcurl4 sqlite3 
-RUN apt-get clean
-RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN apk --update --no-cache add curl sqlite libressl-dev
```

Comment:
Why is it `libressl-dev` and not just `libressl`? :o

Reply:
working with libressl good point